simulator/examples/pacman.rs

```diff
@@ -0,0 +1,68 @@
+//! # Example: Pacman
+//!
+//! An example displaying an animated Pacman.
+
+use embedded_graphics::{
+    pixelcolor::Rgb565, prelude::*, primitives::Circle, primitives::Sector,
+    style::PrimitiveStyleBuilder,
+};
+use embedded_graphics_simulator::{
+    OutputSettingsBuilder, SimulatorDisplay, SimulatorEvent, Window,
+};
+use std::{thread, time::Duration};
+
+// the number of steps of the animation
+const STEPS: i32 = 10;
+
+fn main() -> Result<(), std::convert::Infallible> {
+    // Create a new simulator display with 65x65 pixels.
+    let mut display: SimulatorDisplay<Rgb565> = SimulatorDisplay::new(Size::new(65, 65));
+
+    // Create styles used by the drawing operations.
+    let sector_stroke = PrimitiveStyleBuilder::new()
```

Comment:
These styles apply both stroke and fill:
```suggestion
    let sector_style = PrimitiveStyleBuilder::new()
```

Reply:
OK.